manager/manager.go

```diff
@@ -51,6 +67,8 @@ func BuildRouter() *gin.Engine {
 
 	setMiddlewares(router)
 	createMetaGroup(router, db)
+	createCveGroup(router, db)
```

Comment:
Just a note, in vulnerability-engine we changed to opening/closing DB connection per request, mostly because shared connection didn't work well in case of DB error. It may work better in golang though.

Reply:
gorm.DB is running on database/sql, which is already a connection pool. This also makes sense because gin package serves each request as goroutine, passing single db connection would make some nasty data races. 
(https://gorm.io/docs/generic_interface.html#Connection-Pool , 
 https://pkg.go.dev/database/sql#DB)

Reply:
Ok, I still have some doubts, but let's see how this works.